profiles/resources/default.cfg

```diff
@@ -6,6 +6,9 @@ VulnSearch_____ExploitSearch (Exploits Only)_____http://www.exploitsearch.net/in
 VulnSearch_____ExploitSearch (All)_____http://www.exploitsearch.net/index.php?q=@@@PLACE_HOLDER@@@
 VulnSearch_____NVD (All)_____http://web.nvd.nist.gov/view/vuln/search-results?query=@@@PLACE_HOLDER@@@&search_type=all&cves=on
 VulnSearch_____OSVDB (All)_____http://osvdb.org/search/search?search%5Bvuln_title%5D=@@@PLACE_HOLDER@@@&search%5Btext_type%5D=alltext&search%5Bs_date%5D=&search%5Be_date%5D=&search%5Brefid%5D=&search%5Breferencetypes%5D=&search%5Bvendors%5D=&search%5Bcvss_score_from%5D=7&search%5Bcvss_score_to%5D=10&search%5Bcvss_av%5D=*&search%5Bcvss_ac%5D=*&search%5Bcvss_a%5D=*&search%5Bcvss_ci%5D=*&search%5Bcvss_ii%5D=*&search%5Bcvss_ai%5D=*&kthx=search
+#Descending order of CVEDetails added
+VulnSearch_____CVEDetails_____http://www.google.com/search?sclient=psy&hl=en&site=&source=hp&q=+site:cvedetails.com+inurl:vulnerability-list+inurl:version_id+-inurl:%22year-%22+-inurl:%22opec-%22+intitle:'@@@PLACE_HOLDER@@@'
+VulnSearch_____CVEDetails(Descending order of CVSS)_____http://www.google.com/search?sclient=psy&hl=en&site=&source=hp&q=site:cvedetails.com+inurl:vulnerability-list+inurl:order%3D3+intitle'@@@PLACE_HOLDER@@@'
```

Comment:
Wouldn't it be ``intitle:'@@@PLACE_HOLDER@@@'`` instead of ``intitle'@@@PLACE_HOLDER@@@'``?